emails/_components/Comment.tsx

```diff
@@ -0,0 +1,24 @@
+import { Text } from '@react-email/components';
+
+interface CommentProps {
+  userComment?: string | null;
+  adminComment?: string | null;
+}
+
+export default function Comment({ userComment, adminComment }: CommentProps) {
+  return (
+    <div>
+      {userComment && (
+        <div>
+          <Text>{userComment}</Text>
```

Comment:
curious if we also need title for `User Comment:` as well just like we have the `Admin Comment` below.

Reply:
I was a little unsure on what to do in this case, especially since `userComment` is in the schema for Private Cloud, but not in Public Cloud, and just wanted to have extensibility with this component. But then styling of the Private Cloud and Public Cloud would differ... So, I left out the title/heading for `userComment`

Reply:
it is more for `UI design` rather than the schema changes; I will approve it now and let's see.